### PR TITLE
[Api][Shop][Order][Address] Viewing order

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

Screenshot 2020-09-29 at 13 53 01

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Address.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
```
Comment (AdamKasp) [inline src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Address.xml:14]: Do we need this configuration? Serialization shouldn't be enough? if I see correctly we don't use this `get` method anywhere.

**Review by AdamKasp — APPROVED** [review]

File: features/account/customer_account/viewing_orders_history/viewing_order.feature [hunk 2]
```diff
@@ -16,10 +16,10 @@ Feature: Viewing details of an order
         And for the billing address of "Mazikeen Lilim" in the "Pacific Coast Hwy", "90806" "Los Angeles", "United States"
         And I chose "Free" shipping method with "Cash on Delivery" payment
 
-    @ui
+    @ui @api
     Scenario: Viewing basic information about an order
-        When I view the summary of the order "#00000666"
-        And it should has number "#00000666"
+        When I view the summary of my order "#00000666"
+        Then it should has number "#00000666"
```
Comment (GSadee) [inline features/account/customer_account/viewing_orders_history/viewing_order.feature:22]:
```suggestion
        Then it should have the number "#00000666"
```

File: src/Sylius/Behat/Context/Api/Shop/OrderContext.php [hunk 3]
```diff
@@ -63,6 +65,14 @@ public function __construct(
         $this->sharedStorage = $sharedStorage;
     }
 
+    /**
+     * @When I view the summary of my order :order
+     */
+    public function iSeeMyOrder(OrderInterface $order): void
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Api/Shop/OrderContext.php:71]:
```suggestion
    public function iViewTheSummaryOfMyOrder(OrderInterface $order): void
```

**Review by GSadee — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Tomasz! :tada:

**Pull request merged**